### ISSUE TITLE
Hiding some records from non-Verifiers

### ISSUE THREAD

**Issue by DavidHepper** [BiologicalRecordsCentre/iRecord#1284]
_Feature request_
BDS Recording Scheme has standardised on iRecord as the primary repository for collection, collation and verification. We would like all records to be available to verifiers in iRecord but are having issues with some record providers over permission to import their records to iRecord. Are there any plans - or could there be - to enable the import of records that can only be seen by those with verifier status? During the import this could be flagged either at the whole batch level or the individual row level.

**Comment by johnvanbreda** [comment]
This could be achieved by having a registered website for verifier only records which BDS can import into.

**Comment by DavidHepper** [comment]
Interesting. Would verifiers still be able to see, map and download combined public-iR and 'registered-website' records or would they remain separate?

**Comment by johnvanbreda** [comment]
They would have access to the records in the verification page and potentially for download, but they would not appear in the standard Explore pages.

**Comment by kitenetter** [comment]
@DavidHepper As John says we can implement a closed dataset, and happy to go down this route if you wish. We can make records available to verifiers only, in which case verifiers will be able to download them but they will not be visible on iRecord to non-verifiers, and will also not go to LERCs or NBN Atlas. Is it your intention to have records in iRecord that do not go to the Atlas?

Alternatively records can be made available to both verifiers and LERCs/NBN Atlas, but hidden to non-verifiers on iRecord. However, if they are going to the Atlas I'm not sure why they need hiding on iRecord?

**Comment by DavidHepper** [comment]
Yes, I think a closed dataset would be best, please. These records would mostly be from LERCs and shouldn't go to NBN Atlas. LERCs don't need to see them within our dataset but I should be able to download them so that they can be provided in a controlled way to research organisations such as CEH. It breaks our Open Data principles but even I have to be pragmatic sometimes.
